### PR TITLE
Footer design

### PR DESCRIPTION
**Pull request by nivethikaganeshan**
Created a footer design for the homepage

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -28,3 +27,146 @@ p {
...
+
+.adress {
+    text-align: left; 
+    color: white; 
+    margin-top: 10px; 
+    padding-left: 15px;
+}
+
+.adress h3{
+    text-align: left;
```
Comment (edvardlomo) [inline css/main.css]: All the footer css code can go in a footer.css file, then have it be imported in main.html.

File: no/main.html [hunk 2]
```diff
@@ -16,25 +20,71 @@
...
+			<p>0371 Oslo</p>
+		</div>
+			
+		<div class="footerBottom">
+			<span class="designer">Lyra</span><p class="copyright"> &copy;2024</p>
+		</div>       
+	</footer>	
 </html>
 
+
```
Comment (edvardlomo) [inline no/main.html]: An extra line has been added to the end

**Comment by edvardlomo** [comment]
Other than this, looks good

**Review by sakithabaskaran — APPROVED** [review]
Looks very good! Ready to merge :)

**Pull request merged**